src/rpc/protocol/payloads.rs

```diff
@@ -638,15 +716,44 @@ impl ObTableBatchOperation {
             row_keys,
             Some(columns),
             Some(properties),
+            None,
+            None,
         ));
     }
 
+    /// check the data with corresponding row_keys
+    /// whether meet the filter and execute the insertUp
+    /// if check_exist is true: check if any data meet the filter
+    /// if check_exist is false: check if all data do not meet the filter
```

Comment:
I can't understand the `check_exist` parameter. The comment about it seems confusing.

Reply:
After some discussions, I guess we can hide the `check_exist` parameter, and provide two methods:
```
check_and_upsert_if_exists
check_and_upsert_if_not_exists
```

And it may lead to less confusion.